src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPlugin.groovy

```diff
@@ -77,7 +77,7 @@ class DockerJavaApplicationPlugin implements Plugin<Project> {
             maintainer { dockerJavaApplication.maintainer }
             addFile({ tarTask.archivePath.name }, { '/' })
             entryPoint { determineEntryPoint(project, tarTask) }
-            exposePort { dockerJavaApplication.port }
+            exposePort { dockerJavaApplication.ports }
```

Comment:
Lets use the `getPorts()` method so that groovy won't try and access just the Integer array but instead use the function which takes into account both the Integer array and standalone Integer port.